publisher/server/publisher.go

```diff
@@ -22,9 +22,8 @@ const (
 type Server struct {
 	DockerClient *docker.Client
```

Comment:
:+1: it can be in a separate PR. I think we should make a PR that touches all component to have a common etcd/docker interface.